spec/pactas_itero/api/invoices_spec.rb

```diff
@@ -84,4 +84,32 @@
       expect(request).to have_been_made
     end
   end
+
+  describe '.invoice_download' do
+    it "requests the correct resource" do
```

Comment:
Ich finde da die beiden specs exakt das selbe tun könnte das auch gerne einer sein, was meinst du? Wenn man kein `and` möchte vielleicht `returns the pdf as a string after requesting the correct resource` (das es eine PDF ist finde ich hier btw so oder so noch interessant zur Dokumentation, auch wenn man das wohl schwer testen kann).